### PR TITLE
Add gNMI server

### PR DESCRIPTION
**Pull request by noguchiko**
This PR adds a [gNMI](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md) server application as `north/gnmi`. The server will provide users with a gRPC-based interface for control and management.

 If #56 is merged first, commit 6cfabc805d94ee551f97dbff63f0f98b3c63e83e will be removed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -89,12 +89,13 @@ cmd:
 
 lint:
 	which black && exit `black -q --diff --exclude src/north/snmp/src src | wc -l`
+	which black && exit `black -q --diff --exclude "src/north/snmp/src|src/north/gnmi/goldstone/north/gnmi/proto" src | wc -l`
```
Comment (ishidawataru) [inline Makefile:92]: Do we need to delete the line above?

```
which black && exit `black -q --diff --exclude src/north/snmp/src src | wc -l`
```

**Review by ishidawataru — CHANGES_REQUESTED** [review]
Please update https://github.com/oopt-goldstone/goldstone-mgmt/blob/master/.github/workflows/ci.yaml#L39 and https://github.com/oopt-goldstone/goldstone-mgmt/blob/master/.github/workflows/push.yaml#L45 to build the north-gnmi image in the CI pipeline

File: Makefile (hunk 1 above)
Comment (noguchiko) [reply Makefile:92]: Yes. It's a mistake. I'll delete the line.

Comment (noguchiko) [reply Makefile:91]: done

**Pull request merged**